_plugins/fix_urls.rb

```diff
@@ -12,7 +12,7 @@ def post_read(site)
       Dir.glob(%w[./docker-hub/api/*.yaml ./engine/api/*.yaml]) do |file_name|
         Jekyll.logger.info "    #{file_name}"
         text = File.read(file_name)
-        replace = text.gsub!("https://docs.docker.com", "")
+        replace = text.gsub("https://docs.docker.com", "")
```

Comment:
What does this change do? I think the intent here is to make full URLs (including domain name) relative (no domain name), so that when hosting the docs on a preview domain (or locally) does not link to / use / depend on the live documentation at docs.docker.com.

Reply:
Yes, that should still work. The issue here is that it used to fail if there were no matches.